src/reducers-actions/pendingOrders.ts

```diff
@@ -0,0 +1,79 @@
+import { PendingTxObj } from 'api/exchange/ExchangeApi'
+import { GP_ORDER_TX_HASHES } from 'const'
+
+export const enum ActionTypes {
+  SAVE_PENDING_ORDERS = 'SAVE_PENDING_ORDERS',
+  REMOVE_PENDING_ORDERS = 'REMOVE_PENDING_ORDERS',
+}
+
+interface Actions<T> {
+  type: ActionTypes
+  payload: T
+}
+
+export const savePendingOrdersAction = (payload: {
+  orders: PendingTxObj
+  networkId: number
+}): Actions<{
+  orders: PendingTxObj
+  networkId: number
+}> => ({
+  type: ActionTypes.SAVE_PENDING_ORDERS,
+  payload,
+})
+
+export const removePendingOrdersAction = (payload: {
+  networkId: number
+  pendingTxHash: string
+}): Actions<{
+  networkId: number
+  pendingTxHash: string
+}> => ({
+  type: ActionTypes.REMOVE_PENDING_ORDERS,
+  payload,
+})
+
+export type PendingOrdersState = typeof GP_ORDER_TX_HASHES
+
+export const EMPTY_PENDING_ORDERS_STATE = {
+  1: [],
+  4: [],
+}
```

Comment:
One thing that I realized when testing the pagination is that address is another dimension to be considered.
In this case, when changing addresses the pending txs will still be the same, no?

Reply:
ah FUCKKKKK

Reply:
true

Reply:
ok switched it, good catch